app.py

```diff
@@ -100,3 +104,56 @@ def add_skill():
     data["skill"].append(new_skill)
 
     return jsonify({"id": data["skill"].index(new_skill)})
+
+
+def add_experience():
+    """
+    Add an experience using POST method
```

Comment:
It doesn't look like this function is specific to POST, but rather the fact that we are dealing with a POST request depends on the code above. If that's the case, then the function is making assumption about the context in which it is called, which is in general not desirable (functions are supposed to be atomic and independent on external variables/context). I would change this comment.

Reply:
done